README.md

```diff
@@ -89,3 +92,12 @@ for additional information.
   $ mkdir /some/dir/nexus-data && chown -R 200 /some/dir/nexus-data
   $ docker run -d -p 8081:8081 --name nexus -v /some/dir/nexus-data:/nexus-data sonatype/docker-nexus3
   ```
+
+## Getting Help
+
+Looking to contribute to our Docker image but need some help? There's a few ways to get information or our attention:
+
+* File a public issue [here on GitHub](https://github.com/sonatype/docker-nexus3/issues)
```

Comment:
We either need to turn off issues completely and point people to JIRA (similar to nexus-public) or roll with issues. I think for users it's easier to have issues on, and since I'm actively checking them, I don't mind, especially for a project that isn't Nexus per se.